docs/testing/debug-add-ins-using-f12-developer-tools-on-windows-10.md

```diff
@@ -11,15 +11,18 @@ There are developer tools outside of IDEs available to help you debug your add-i
 
 The tool that you use depends on whether the add-in is running in Microsoft Edge or Internet Explorer. This is determined by the version of Windows 10 and the version of Office that are installed on the computer. To determine which browser is being used on your development computer, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). 
 
-
 > [!NOTE]
 > The instructions in this article cannot be used to debug an Outlook add-in that uses Execute Functions. To debug an Outlook add-in that uses Execute Functions, we recommend that you attach to Visual Studio in script mode or to some other script debugger.
```

Comment:
If the "Enable debugging for add-in commands and UI-less code" works for Outlook add-ins using execute functions, then this note is not needed, right?

Reply:
Hi @ElizabethSamuel-MSFT I confirmed that this note still applies, so left it in. Thanks!